common/utils/src/main/scala/org/apache/spark/internal/LogKey.scala

```diff
@@ -110,7 +121,8 @@ object LogKey extends Enumeration {
   val REDUCE_ID = Value
   val RELATION_NAME = Value
   val REMAINING_PARTITIONS = Value
-  val REMOTE_ADDRESS = Value
```

Comment:
I found that the LogKey `REMOTE_ADDRESS` and `RPC_ADDRESS` have the same meaning, so I removed `REMOTE_ADDRESS` from `LogKey` and only kept `RPC_ADDRESS`